CreeDictionary/search_quality/sample.csv

```diff
@@ -208,6 +209,9 @@ talks,pîkiskwâtêw,,,,
 sees it,wâpahtam,,,,
 sees him,wâpamêw,,,,
 sees her,wâpamêw,,,,
+see it,wâpahtam,,,,
```

Comment:
[pahkwêwsikan](https://itwewina.altlab.app/word/pahkw%C3%AAsikan/) is a 🧑🏽 _awa_ ("animate") noun, but "bread" is an "it" in English

```suggestion
see it,wâpahtam,wâpamêw,,,
```

Reply:
I’m unfamiliar with this, so let me make sure I understand: Cree animacy != English itness, so ‘see it’ can refer to both animate and inanimate things, so return both the animate and inanimate verb?

Do you mean

```
--- a/CreeDictionary/search_quality/sample.csv
+++ b/CreeDictionary/search_quality/sample.csv
@@ -206,10 +206,10 @@ wants him,nitawêyimêw,,,,
 wants her,nitawêyimêw,,,,
 wants them,nitawêyimêw,,,,
 talks,pîkiskwâtêw,,,,
-sees it,wâpahtam,,,,
+sees it,wâpahtam,wâpamêw,,,
 sees him,wâpamêw,,,,
 sees her,wâpamêw,,,,
-see it,wâpahtam,,,,
+see it,wâpahtam,wâpamêw,,,
 see him,wâpamêw,,,,
 see her,wâpamêw,,,,
 lives at,wîkiw,,,,
```

?

Reply:
Yes!

Animacy in Cree is a bit like gender in German. Sometimes it makes intuitive sense like "die Frau", but sometimes it's aribrary like "die Brücke". Which is why I like calling them "awa"/"ôma" words instead of "animate"/"inanimate". Bread is "awa" and socks are "awa". But water is "ôma".

I think most nouns that are, in a real life sense, animate (having souls and agency) are "awa" but _some_ things that lack souls (like socks) become "awa" too.

Sorry! Language is hard!